guides/common/modules/con_upgrading-el-on-project-or-proxy.adoc

```diff
@@ -4,7 +4,7 @@
 {Project} and {SmartProxy} are supported on both {EL} 8 and {EL} 9.
 You can upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9.
 
-There are two methods to upgrade your {Project} or {SmartProxy} from {EL} 8 to {EL} 9:
+You can use the following methods to upgrade your {Project} or {SmartProxy} from {EL} 8 to {EL} 9:
```

Comment:
Not a writer, but this now uses "You can", just like the sentence before it. In fact, this sentence is now very close to the one before it. Should they be merged?

Reply:
Shame on me for not checking the context! That's how issues like the one I've set out to fix occur in the first place. Thanks for the catch.